src/bindings/python/src/openvino/torch/requirements.txt

```diff
@@ -0,0 +1 @@
+torch>=2.0.0
```

Comment:
Why do we need it?

Reply:
I just added the requirements file because if we import openvino.torch, it fails if torch is not installed which is the expected behaviour. If user wants to use torch.compile, they will anyway have torch installed but i added this and wanted to add documentation for this feature, where we can ask them to just use the requirements file. We can remove it if unecessary.

Reply:
I will remove it